patches/@rnmapbox+maps+10.1.11.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/@rnmapbox/maps/android/src/main/java/com/rnmapbox/rnmbx/components/camera/RNMBXCamera.kt b/node_modules/@rnmapbox/maps/android/src/main/java/com/rnmapbox/rnmbx/components/camera/RNMBXCamera.kt
+index dbd6d0b..1d043f2 100644
+--- a/node_modules/@rnmapbox/maps/android/src/main/java/com/rnmapbox/rnmbx/components/camera/RNMBXCamera.kt
++++ b/node_modules/@rnmapbox/maps/android/src/main/java/com/rnmapbox/rnmbx/components/camera/RNMBXCamera.kt
+@@ -188,7 +188,7 @@ class RNMBXCamera(private val mContext: Context, private val mManager: RNMBXCame
+ 
+     private fun setInitialCamera() {
+         mDefaultStop?.let {
+-            val mapView = mMapView!!
++            val mapView = mMapView ?: return
```

Comment:
not seeing an upstream PR for this one, is it critical? Please either remove the patch or open an upstream PR and include the link in the PR description